### ISSUE TITLE
gspca_kinect2: probe failed with error -22

### ISSUE THREAD

**Issue by BOFH2k** [yoshimoto/gspca-kinect2#7]
Similar probing issue to #3 (even after adding the CFLAG from #6, fixing the [Makefile](https://github.com/yoshimoto/gspca-kinect2/files/4708274/Makefile.txt) and properly inserting the [modules](https://github.com/yoshimoto/gspca-kinect2/files/4708275/insmod.txt)) with kernel 5.4.0-33-generic in Ubuntu 20.04:
```
[ 2876.003031] gspca_main: video_register_device err -22
[ 2876.003076] gspca_kinect2: probe of 11-2.1:1.0 failed with error -22
```
Details:
[dmesg.txt](https://github.com/yoshimoto/gspca-kinect2/files/4708273/dmesg.txt)

The Kinect v2 for Windows hardware work fine in Windows 10 and the `simple` [libk4w2](https://github.com/yoshimoto/libk4w2) libk4w2 test script runs fine as well, while the `opengl` test script fails:
```
libk4w2/src/driver_v4l2.c:50 Cannot identify '/dev/video0'; No such file or directory
```

**Comment by thamma** [comment]
Thanks for posting.

After loading several kernel modules (including videobuf2-v4l2, videodev, videobuf2_vmalloc, videobuf2-common, videobuf2-memops), and successfully building using said make commands, I get error -22 as well on 5.6.14 on Arch Linux.

**Comment by grandchild** [comment]
I managed to fix this :)

**Comment by BOFH2k** [comment]
I can attest to your branch solving the probe issue.
Could you actually use the webcam then?

**Comment by grandchild** [comment]
Yes. See, again, [here](https://github.com/yoshimoto/gspca-kinect2/pull/8#issuecomment-642292281) and above.

**Comment by grandchild** [comment]
And maybe leave the issue open, since the PR is not actually merged...
